### DISCUSSION TITLE
changing indent_size for yaml files from 4 to 2 spaces?

### DISCUSSION THREAD

**Discussion by mindpilot** [ddev/ddev#5399] [General]
`ddev config` commands are converting 2 spaces to 4 spaces on all indented lines. 
I found a related posts/PR, but no answer it there is a way to change this behavior or this can be considered as a bug.

https://github.com/ddev/ddev/pull/4745
https://github.com/ddev/ddev/blob/master/.editorconfig

ddev version v1.22.1
Mac OS: 13.6 (22G120)

**Comment by rfay** [comment]
Can you please give an example? The linked issue was about .editorconfig and markdown files, which doesn't have to do with yaml. 

Please give a complete example of what you're seeing.

**Reply by mindpilot** [reply]
Hi @rfay thanks for your quick response.
After issuing any `ddev config` command, all indents are increased by 2 spaces.

![ddev-config-indent-change](https://github.com/ddev/ddev/assets/3437392/ddd126ff-0d3a-4ff0-a990-82605d41a750)

**Reply by rfay** [reply]
When you do a `ddev config <sometjing>` it rewrites the config. Although it's possible to change the indent (https://pkg.go.dev/gopkg.in/yaml.v3#Encoder.SetIndent) I wouldn't think that the 4-space revolutionaries would be pleased if we changed it to 2 to satisfy the counter-revolutionaries!

**Reply by mindpilot** [reply]
;-) All right. If 4 is the intended amount, everything is fine. Thanks!
